external/proto/status.proto

```diff
@@ -16,3 +16,21 @@ message Response {
    map<string, onet.Status> status = 1;
   optional network.ServerIdentity serveridentity = 2;
 }
+
+// Connectivity is sent by a client to check the connectivity of a given roster. The Time must
+// be within 2 minutes of the server's time. The signature must be a schnorr-signature using the
+// private conode-key on the following message:
+//   sha256( bytes.LittleEndian.PutUInt64(Time) | protobuf.Encode(List[0]) |
+//   protobuf.Encode(List[1])... )
+message Connectivity {
```

Comment:
Can we name this `ConnectivityRequest` ? So we can see in the code that's a message right away.

Reply:
Most of the other API endpoints (excluded the status-request itself) have the following naming convention:

- service-method `Name`
- api-method `Client.Name`
- request-structure `Name`
- reply-structure `NameReply` or `NameResponse`

That should probably be part of some _Cleanup-task_ for v4... So whatever you chose, all the other APIs should follow. The above naming-scheme is, as far as I know, used in the majority of API calls over the websockets.